### ISSUE TITLE
More info on dense landmarks

### ISSUE THREAD

**Issue by jez-moxmo** [microsoft/SynthMoCap#11]
Hi, thank you for use of the dataset. I am trying to replicate the training code and was wondering if you could add the indecies for the dense 2d landmarks. Is it trained on projected image coords or are they normalized? Are you able to point me in the right direction for landmark loss? Thank you kindly.

**Comment by friggog** [comment]
We can't release the dense landmark set, but you can easily define your own - we aimed to capture extremes of the surface and give a reasonably even coverage in terms of density, I suspect any sensible looking set of dense landmarks will prove similarly effective. The landmark loss is given in the paper (eq 1) - just standard Gaussian negative log likelihood loss (GNLL). The landmark locations are normalized within the ROI
